.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Hello - thanks for reporting an issue with Habitat.
+
+In order to help us troubleshoot the issue, please be sure to include the following information if applicable:
+
+- [ ] The OS (including version) where you are running any of the Habitat commands
+- [ ] Debug/backtrace of the command you are trying to run. You can set the following environment variables before running the `hab` command to generate a trace:
+
+		RUST_LOG=debug RUST_BACKTRACE=1 hab ...
+- [ ] Current Habitat environment variables. These can be gathered using:
+
+		env | grep HAB
```

Comment:
fixed and included a note in bold/italics not to include that env var. Do you think there is a better way to collect env vars?